src/mlpack/methods/ann/activation_functions/gelu_function.hpp

```diff
@@ -0,0 +1,93 @@
+/**
+ * @file gelu_function.hpp
+ * @author Himanshu Pathak
+ *
+ * Definition and implementation of the Gaussian Error Linear Unit (GELU)
+ * function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_GELU_FUNCTION_HPP
+#define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_GELU_FUNCTION_HPP
+# define PI 3.141592653589793238462643383279502884L /* pi */
```

Comment:
You can use the already defined `M_PI` variable.

Reply:
Thanks for the review @zoq . I added all suggested changes.